src/llvm-late-gc-lowering.cpp

```diff
@@ -461,6 +461,23 @@ static std::pair<Value*,int> FindBaseValue(const State &S, Value *V, bool UseCac
             fld_idx = IdxOp->getLimitedValue(INT_MAX);
             CurrentV = EEI->getVectorOperand();
         }
+        else if (auto LI = dyn_cast<LoadInst>(CurrentV)) {
+            if (auto PtrT = dyn_cast<PointerType>(LI->getType())) {
+                if (PtrT->getAddressSpace() == AddressSpace::Loaded) {
+                    CurrentV = LI->getPointerOperand();
+                    if (getValueAddrSpace(CurrentV) == 0) {
```

Comment:
should this be `if (!isSpecialPtr(getValueAddrSpace(CurrentV)))`, or is the idea that we only want to allow the base pointer to be either Tracked or None

Reply:
We don't really have non-gc address spaces in a target that can do GC, so I don't think it matters too much, but I'll change it.